frontend/src/scenes/App.tsx

```diff
@@ -141,18 +141,21 @@ function _App(): JSX.Element | null {
 
                         {!featureFlags['hide-billing-toolbar'] && <BillingToolbar />}
                         {featureFlags['navigation-1775'] ? <BackTo /> : null}
-                        {currentTeam && !currentTeam.completed_snippet_onboarding && !sceneConfig.hideDemoWarnings && (
-                            <Alert
-                                type="warning"
-                                style={{ marginTop: featureFlags['navigation-1775'] ? '1rem' : 0 }}
-                                message={
-                                    <>
-                                        You haven't sent any events to this project yet. Grab{' '}
-                                        <Link to="/project/settings">a snippet or library</Link> to get started!
-                                    </>
-                                }
-                            />
-                        )}
+                        {currentTeam &&
```

Comment:
Do we want to keep this warning around? It will cause issues with the other DemoWarning code.

Reply:
I think we want to keep it, but we might need to adjust it. Consider this case: you create a second real-data project, but don't ingest any events. We won't show the new demo warning, but this one might be helpful. Maybe worth doing this in a separate PR?